tool/site/client-react/src/components/claim/createDispenseClaim.ts

```diff
@@ -161,14 +162,18 @@ function createClaimItemDetail(
   }
 
   const finalMedicationDispense = medicationDispenses[medicationDispenses.length - 1]
+  const finalItemStatus = finalMedicationDispense.type
+  const fullyDispensed = finalItemStatus.coding[0].code === LineItemStatus.DISPENSED
 
   return {
     extension: claimItemDetailExtensions,
     sequence,
     productOrService: medicationRequest.medicationCodeableConcept,
-    modifier: [finalMedicationDispense.type],
+    modifier: [finalItemStatus],
     quantity: medicationRequest.dispenseRequest.quantity,
-    subDetail: [createClaimItemDetailSubDetail(1, medicationDispenses, productFormValues)]
+    subDetail: fullyDispensed
```

Comment:
We could instead have made this conditional on `productFormValues` being truthy, but I thought this was probably clearer